.github/workflows/build.sh

```diff
@@ -37,6 +37,10 @@ else
 fi
 
 mdbook build -d "$dest_dir"
+
+# Disable the redbox button in built versions of the course
+echo '// disabled' > "${dest_dir}/html/theme/redbox.js"
```

Comment:
I think it will be easiest to just erase this JS file in the built version. If this looks good to you and @mgeisler then I think we can merge!

Reply:
Sorry, I don't know,I am not sure about it.Lets wait for the review of @mgeisler .

Reply:
I think this is a nice approach, thanks @djmitche.

Since the feature is entirely triggered by the configuration in `book.toml`, a different solution would be to manipulate this file. This can be done with environment variables which override the `book.toml` file.

Unfortunately, it's not super easy to remove the `redbox.js` and `redbox.css` entries from the two lists... I was playing with a solution that uses [`toml`](https://github.com/gnprice/toml-cli) and [`jq`](https://jqlang.github.io/jq/) — here is how to get the other JS files:

```
toml get book.toml 'output.html.additional-js' | jq 'map(select(test("redbox") | not))'
```

that value could be put back into a `MDBOOK_OUTPUT__HTML__ADDITIONAL_JS` environment variable in `build.sh` and this will also disable the button (the same should then be done for the `additional-css` key).

This is probably overkill and I like your simple solution very much.